apps/ui/src/Pages/HomePage/HomePage.tsx

```diff
@@ -60,7 +75,10 @@ const HomePage: React.FC = () => {
             </ToggleButtonGroup>
           </Box>
           {selectedTab === 'communities' && (
```

Comment:
consider put 'communities' in a const

Reply:
it's part of `selectedTab`'s type so it can be either equal to 'communities' or 'rides' nothing else